Sources/SwiftSDKGenerator/Generator/SwiftSDKGenerator+Copy.swift

```diff
@@ -48,36 +48,42 @@ extension SwiftSDKGenerator {
           )
         }
 
-        let sdkUsrLib64Path = sdkUsrPath.appending("lib64")
-        try await generator.copyFromDockerContainer(
-          id: containerID,
-          from: FilePath("/usr/lib64"),
-          to: sdkUsrLib64Path
-        )
-        try await createSymlink(at: pathsConfiguration.sdkDirPath.appending("lib64"), pointingTo: "./usr/lib64")
+        if case let containerLib64 = FilePath("/usr/lib64"),
+           try await generator.doesPathExist(containerLib64, inContainer: containerID) {
+          let sdkUsrLib64Path = sdkUsrPath.appending("lib64")
+          try await generator.copyFromDockerContainer(
+            id: containerID,
+            from: containerLib64,
+            to: sdkUsrLib64Path,
+            failIfNotExists: false
+          )
+          try await createSymlink(at: pathsConfiguration.sdkDirPath.appending("lib64"), pointingTo: "./usr/lib64")
 
-        if case .rhel = self.versionsConfiguration.linuxDistribution {
-          // `libc.so` is a linker script with absolute paths on RHEL, replace with a relative symlink
-          let libcSO = sdkUsrLib64Path.appending("libc.so")
-          try await removeFile(at: libcSO)
-          try await createSymlink(at: libcSO, pointingTo: "libc.so.6")
+          if case .rhel = self.versionsConfiguration.linuxDistribution {
+            // `libc.so` is a linker script with absolute paths on RHEL, replace with a relative symlink
+            let libcSO = sdkUsrLib64Path.appending("libc.so")
+            try await removeFile(at: libcSO)
+            try await createSymlink(at: libcSO, pointingTo: "libc.so.6")
+          }
         }
 
         try await generator.createDirectoryIfNeeded(at: sdkUsrLibPath)
-        var subpaths = ["clang", "gcc", "swift", "swift_static"]
+        var subpaths = [("clang", true), ("gcc", true), ("swift", true), ("swift_static", true)]
```

Comment:
Let's make clear what tuple elements stand for at the place of declaration. Alternatively, this would benefit from a separate struct that has explicitly named fields.

```suggestion
        var subpaths: [(subpath: String, failIfNotExists)] = [
          ("clang", true), ("gcc", true), ("swift", true), ("swift_static", true)
        ]
```